packages/p2p/src/components/app.jsx

```diff
@@ -246,7 +258,10 @@ class App extends React.Component {
     };
 
     updateP2pNotifications = notifications => {
-        const notification_count = notifications.filter(notification => notification.is_seen === false).length;
+        const unseen_notification = notifications.filter(notification => notification.is_seen === false);
```

Comment:
Since there are multiple notifications it'll be clearer to use a plural, e.g.

```suggestion
        const unseen_notifications = notifications.filter(notification => notification.is_seen === false);
```